### DISCUSSION TITLE
Homebrew tap doesn't support Gitlab url

### DISCUSSION THREAD

**Discussion by aminoor-alam** [renovatebot/renovate#10974] [Request Help]
Currently, Renovate only supports Github url - is there any plan to support Gitlab url? currently outputs as null:

https://github.com/renovatebot/renovate/blob/main/lib/manager/homebrew/extract.ts#L64

**Comment by viceice** [comment]
No plan yet

**Comment by rarkins** [comment]
Please raise a feature request issue, ideally with a minimal reproduction repo to test against.
